artifact/image/keys_test.go

```diff
@@ -65,58 +67,29 @@ func signatureTest(t *testing.T, privateKey []byte) {
 	// much, since the header will be placed on it by the image
 	// tool.
 
-	simpleName := path.Join(tmpdir, "simple.bin")
-	hashedName := path.Join(tmpdir, "simple-hashed.bin")
-	signedName := path.Join(tmpdir, "simple-signed.bin")
-	keyName := path.Join(tmpdir, "private.pem")
-
-	tmp := make([]byte, 256)
-	for i := 0; i < len(tmp); i++ {
-		tmp[i] = byte(i & 0xFF)
-	}
-	err = ioutil.WriteFile(simpleName, tmp, 0644)
-	if err != nil {
-		t.Fatal(err)
+	body := make([]byte, 256)
+	for i := 0; i < len(body); i++ {
+		body[i] = byte(i & 0xFF)
```

Comment:
Why is doing an `&` here different than just doing `= byte(i)`?

Reply:
Good spot.  Casting to `byte` will perform the desired truncation automatically.